utils/create_srg_export.py

```diff
@@ -220,6 +444,17 @@ def get_rule_json(json_path: str) -> dict:
     return rule_json
 
 
+def add_variables(env_yaml, policy):
+    for control in policy.controls:
+        for rule in control.selections:
+            if rule.startswith('var_'):
+                parts = rule.split('=')
```

Comment:
Selected item can be a rule, but here it is obvious that it can also be a variable or a refinement. Therefore, the name `rule` is misleading.
Next, you identify variables using the assumption that they start with `var_`, which is incorrect - there is e.g. the `sshd_approved_macs` variable.
However, you can use `ssg.build_yaml.SelectionHandler` and its `apply_selection` that identifies all variables for you.